contracts/test/forkMainnet/AMMWrapper/CollectFee.t.sol

```diff
@@ -34,12 +34,12 @@ contract TestAMMWrapperCollectFee is TestAMMWrapper {
         bytes memory sig = _signTrade(userPrivateKey, order);
         bytes memory payload = _genTradePayload(order, feeFactor, sig);
 
-        BalanceSnapshot.Snapshot memory ammWrapperMakerAsset = BalanceSnapshot.take(address(ammWrapper), order.makerAssetAddr);
+        BalanceSnapshot.Snapshot memory feeCollectorMakerAsset = BalanceSnapshot.take(address(feeCollector), order.makerAssetAddr);
```

Comment:
Maybe we could remove `address()` wrap since `feeCollector` is already an address value

Reply:
Fixed in d281833